WindowsServerDocs/storage/storage-spaces/delimit-volume-allocation.md

```diff
@@ -5,7 +5,7 @@ ms.manager: eldenc
 ms.technology: storage-spaces
 ms.topic: article
 author: cosmosdarwin
-ms.date: 02/28/2018
+ms.date: 03/03/2018
 ---
 
 # Delimit the allocation of volumes in Storage Spaces Direct
```

Comment:
As we discussed, consider renaming the feature "consolidated allocation", etc.

Reply:
Hm, I think either would be fine; let's keep it for now?